Add example for Kubernetes deployment

thebillkidy opened the pull request:
# Description

Adding example for Kubernetes deployment

## Issue reference

#30 as well as #27 

## Checklist

Please make sure you've completed the relevant tasks for this PR, out of the following list:

* [x] Code compiles correctly
* [ ] Created/updated tests
* [x] Extended the documentation

youngbupark commented:
@thebillkidy Sorry for the late response. Thanks for your contribution. In 0.7.0, we changed gRPC proto files a lot and added the script to generate proto stub and client codes. So the current code will not work once we release 0.7.0 release. 

Here is my comments on the PR:
1. Please wait to merge this pr until 0.7.0 dapr is released
2. Use `dapr-client` instead of generating proto client codes while building docker image
3. @pruthvidhodda already merged the proto client [generation script](https://github.com/dapr/python-sdk/blob/master/protobuf.sh)

thebillkidy commented:
> @thebillkidy Sorry for the late response. Thanks for your contribution. In 0.7.0, we changed gRPC proto files a lot and added the script to generate proto stub and client codes. So the current code will not work once we release 0.7.0 release.
> 
> Here is my comments on the PR:
> 
> 1. Please wait to merge this pr until 0.7.0 dapr is released
> 2. Use `dapr-client` instead of generating proto client codes while building docker image
> 3. @pruthvidhodda already merged the proto client [generation script](https://github.com/dapr/python-sdk/blob/master/protobuf.sh)

Perfect, thanks a lot! Will wait until release and then update :)

yaron2 commented:
Would love to see this going forward now.

youngbupark commented:
@shalabhms Seems like we didn't publish the latest dapr-client sdk. Can you please publish it?

JFYI - I will set up CI/CD to publish it automatically in this milstone. 

cc/ @amanbha

thebillkidy commented:
I blocked some time for this tomorrow, will finalize it then if the dapr-client sdk was pushed correctly :)

amanbha commented:
> @shalabhms Seems like we didn't publish the latest dapr-client sdk. Can you please publish it?
> 
> JFYI - I will set up CI/CD to publish it automatically in this milstone.
> 
> cc/ @amanbha

I have uploaded the new python package

youngbupark commented:
> > @shalabhms Seems like we didn't publish the latest dapr-client sdk. Can you please publish it?
> > JFYI - I will set up CI/CD to publish it automatically in this milstone.
> > cc/ @amanbha
> 
> I have uploaded the new python package

@amanbha Thanks! :)

thebillkidy commented:
@youngbupark example updated for Dapr v0.7.1 😊

P.S. Thanks a lot for updating the other examples as well!

youngbupark commented on examples/kubernetes/invoke-simple/Proto/dapr.proto:
```diff
@@ -0,0 +1,120 @@
+
+package dapr;
+
+import "google/protobuf/any.proto";
```
Do we still need this proto file?

thebillkidy replied:
Nope, removed it :)

youngbupark commented on examples/kubernetes/invoke-simple/Scripts/generate-proto.sh:
```diff
@@ -0,0 +1,13 @@
+if [ -z $1 -o -z $2 ]; then
+  echo "Usage: ./generate-proto.sh <LOCATION_PROTO> <OUTPUT_FOLDER>"
```
Nit: we can remove this script now because there is the script in root.
